packages/fluentui/projects-test/src/index.ts

```diff
@@ -10,7 +10,8 @@ async function performTest() {
     await rollup();
     await typings();
 
-    await createReactApp();
+    // Skip it till node update
+    // await createReactApp();
```

Comment:
node update is already in, we can reenable

```suggestion
    await createReactApp();
```